Process.c

```diff
@@ -244,6 +244,11 @@ void Process_outputRate(RichString* str, char* buffer, size_t n, double rate, in
    }
 }
 
+void Process_printLeftAlignedField(RichString* str, int attr, const char* content, unsigned int width) {
+   int c = RichString_appendnWide(str, attr, content, MINIMUM(width, strlen(content)));
```

Comment:
strlen will not account for wide chars (it essentially counts bytes)
So testcase:
```bash
echo -e '#!/bin/bash\nsleep 600' > 你好吗 && chmod +x 你好吗 && ./你好吗
```
and run htop with COMM as a visible column

Reply:
Ref: https://github.com/htop-dev/htop/blob/master/RichString.c#L53